en/docs/Learn/DesignAPI/MockAPI/deploy-and-test-mock-apis.md

```diff
@@ -1,32 +1,39 @@
 # Deploy and Test Prototype APIs
 
-## Introduction
-An **API prototype** is created for the purpose of early promotion and testing. You can deploy a new API or a new version of an existing API as a prototype. It gives subscribers an early implementation of the API that they can try out without a subscription or monetization, and provide feedback to improve. After a period of time, publishers can make changes that the users request and publish the API.
+Generally, you would need to create APIs with inline scripts for testing purposes. You can deploy a new API or a new version of an existing API as a prototype. Thereby, this provides subscribers an early implementation of the API that they can try out and test without a subscription or monetization, and provide feedback to improve the API. After a period of time, the publishers can make changes that the users requested and publish the API.
```

Comment:
Seems we are indicating inline scripting as more common with this sentence. Its really up to the end user on how they would like to prototype and one is not more common than the other